Refresh.GameServer/Endpoints/Game/Levels/LeaderboardEndpoints.cs

```diff
@@ -25,6 +25,27 @@ public Response PlayLevel(RequestContext context, GameUser user, GameDatabaseCon
         return OK;
     }
 
+    [GameEndpoint("scoreboard/developer/{id}", Method.Get, ContentType.Xml)]
+    public SerializedLeaderboardResponse GetDeveloperScores(RequestContext context, GameUser user, GameDatabaseContext database, int id)
+    {
+        //TODO
+        return new SerializedLeaderboardResponse(new List<SerializedPlayerLeaderboardResponse>());
+    }
+
+    [GameEndpoint("scoreboard/user/{id}", Method.Get, ContentType.Xml)]
+    public SerializedLeaderboardResponse GetUserScores(RequestContext context, GameUser user, GameDatabaseContext database, int id)
+    {
+        //TODO
+        return new SerializedLeaderboardResponse(new List<SerializedPlayerLeaderboardResponse>());
+    }
+
+    [GameEndpoint("scoreboard/developer/{id}", ContentType.Xml, Method.Post)]
+    public Response SubmitDeveloperScore(RequestContext context, GameUser user, GameDatabaseContext database, int id, SerializedScore body)
+    {
+        //TODO
+        return new Response(SerializedScoreLeaderboardList.FromSubmittedEnumerable(new List<ScoreWithRank>()), ContentType.Xml);
+    }
+
```

Comment:
I think these should definitely be implemented, at least for user levels. Also `scoreboard/user/{id}` might be a duplicate? This seems a little strange.

Reply:
Oops, I was confusing this for topscores/user/id/{type}. My bad.